### ISSUE TITLE
Incomplete certificate chains on HTTPS

### ISSUE THREAD

**Issue by michaelwood** [ThreeSixtyGiving/datastore#81]
We've had a few publishers recently where the `datagetter` hasn't fetched their data due to inability to verify the certificate. This is caused by the lack of full certificate chain being installed on the publisher's website.

Often when we test the links they do load correctly in Firefox/Chrom* due to both browser vendors shipping many many more trusted certificates than done so in the Linux distros (Ubuntu in our case)  that fill in the gaps in the trust chain. 

There are two ways we could get those certificates, simply install firefox on the datastore and periodically copy the certificates to the right path (system? or modify `requests` to look in a certain place?) or use something like https://curl.se/docs/caextract.html , the latter is neater but trusting that source seems more of a leap than trusting the Linux Distro's browser package..

**Comment by Bjwebb** [comment]
Looks like [requests uses](https://docs.python-requests.org/en/master/user/advanced/#ca-certificates) https://pypi.org/project/certifi/ rather than the the distro ca certs. So could be worth 1. seeing if updating certifi makes any difference, 2. pointing it at the distro certs to see if that helps.

**Comment by michaelwood** [comment]
> Looks like [requests uses](https://docs.python-requests.org/en/master/user/advanced/#ca-certificates) https://pypi.org/project/certifi/ rather than the the distro ca certs. So could be worth 1. seeing if updating certifi makes any difference, 2. pointing it at the distro certs to see if that helps.

Yeah I did a quick test and it still fails unfortunately even with the latest certifi

```python
`>>> certifi.__file__
'/tmp/.ve/lib/python3.8/site-packages/certifi/__init__.py'
>>> certifi.__version__
'2020.12.05' ## This is the latest version
>>> requests.__file__
'/tmp/.ve/lib/python3.8/site-packages/requests/__init__.py'

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/.ve/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/tmp/.ve/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/tmp/.ve/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/tmp/.ve/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/tmp/.ve/lib/python3.8/site-packages/requests/adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='www.spiritof2012.org.uk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1123)')))
```

Unless there is something else funky going on...

**Comment by Bjwebb** [comment]
In that case, I think you're right, you need to grab the browser certs.
